pyroomacoustics/doa/music.py

```diff
@@ -84,7 +89,15 @@ def _process(self, X):
         identity[:, list(np.arange(self.M)), list(np.arange(self.M))] = 1
         cross = identity - np.matmul(Es, np.moveaxis(np.conjugate(Es), -1, -2))
         self.Pssl = self._compute_spatial_spectrumvec(cross)
+        if self.frequency_normalization:
+            self._apply_frequency_normalization()
         self.grid.set_values(np.squeeze(np.sum(self.Pssl, axis=1) / self.num_freq))
+        
+    def _apply_frequency_normalization(self):
+        """
+        Normalize the MUSIC pseudo-spectrum per frequency bin
+        """
+        self.Pssl = self.Pssl / np.max(self.Pssl, axis=0)[None,...]
```

Comment:
This could be written slightly more explicitly as `np.max(self.Pssl, axis=0, keepdims=True)`.

Reply:
Hi, thank you for the comments.
I totally agree with 2. and the proposed modification in line 100.

However, I also see a good reason to use frequency_normalization=True as the default behavior.

Obviously, the advantage of frequency_normalization=False is backward compatibility.

From a user's perspective, I think frequency_normalization=True is beneficial. Often, users don't tweak every parameter of a library and just use the standard parameters. Considering that, I think frequency_normalization=True is advantageous due to its much better performance.

Reply:
I understand where you are coming from! From my point of view, there are two important reasons to keep the default behavior to `frequency_normalization=False`

1. Backward compatibility
2. This is the *expected* implementation of MUSIC, i.e., how it is described in most textbooks (that I am aware of)

What about creating a new thin-wrapper class around the `MUSIC` class, e.g., `NormMUSIC`, where normalization is active by default ? I think this would be a good middle ground solution. By having an extra class, the existence of this option would be prominently featured in the doc. Then, the `MUSIC` class would still be backward compatible and do what one expect by default.

To further make the point, what about creating an example script comparing `MUSIC` to `NormMUSIC` in terms of accuracy ? Then, we can point to it in the doc so that people use `NormMUSIC` first.

Reply:
Hi, thanks you for your suggestion and sorry for the late answer.
I agree with you, an extra class "NormMUSIC" is the better solution. 
I will implement it soon.

Reply:
Fantastic! Looking forward to it! 👍